components/zoom_admin/zoom_admin.app.mjs

```diff
@@ -0,0 +1,374 @@
+      optional: true,
+      options: zoomCountries,
+    },
+    pageSize: {
+      type: "integer",
+      label: "Page Size",
+      description: "The number of records returned within a single API call. Defaults to `30`",
+      optional: true,
+      min: 1,
+      max: 300,
+    },
+    pageNumber: {
+      type: "integer",
+      label: "Page Number",
+      description: "The page number of the current page in the returned records.",
+      optional: true,
+      min: 1,
+    },
+    nextPageToken: {
+      type: "string",
+      label: "Next Page Token",
+      description: "The next page token is used to paginate through large result sets. A next page token will be returned whenever the set of available results exceeds the current page size. The expiration period for this token is 15 minutes.",
+      optional: true,
+    },
+    webinar: {
+      type: "string",
+      label: "Webinar",
+      optional: true,
+      description: "The Webinar ID",
+      async options({
+        prevContext,
+        page,
+      }) {
+        if (!prevContext.nextPageToken && page > 0) {
+          return [];
+        }
+
+        const {
+          webinars,
+          next_page_token,
+        } = await this.listWebinars({
+          nextPageToken: prevContext.nextPageToken,
+          pageSize: 2,
+        });
+        if (!webinars.length) {
+          return [];
+        }
+        const rawOptions = webinars.map((w) => ({
+          label: w.topic,
+          value: {
+            label: w.topic,
+            value: w.id,
+          },
+        }));
+        const options = sortBy(rawOptions, [
+          "label",
+        ]);
+
+        return {
+          options,
+          context: {
+            nextPageToken: next_page_token,
+          },
+        };
+      },
+    },
+  },
+  methods: {
+    _apiUrl() {
+      return "https://api.zoom.us/v2";
+    },
+    _getHeaders() {
+      return {
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+        "Authorization": `Bearer ${this.$auth.oauth_access_token}`,
+        "user-agent": "@PipedreamHQ/pipedream v0.1",
+      };
+    },
+    _getAxiosParams(opts = {}) {
+      return {
+        ...opts,
+        url: this._apiUrl() + opts.path,
+        headers: this._getHeaders(),
+      };
+    },
+    async _makeRequest(opts) {
+      if (!opts.headers) opts.headers = {};
+      opts.headers["Accept"] = "application/json";
+      opts.headers["Content-Type"] = "application/json";
+      opts.headers["Authorization"] = `Bearer ${this.$auth.oauth_access_token}`;
+      opts.headers["user-agent"] = "@PipedreamHQ/pipedream v0.1";
+      const { path } = opts;
+      delete opts.path;
+      // eslint-disable-next-line multiline-ternary
+      opts.url = `${this._apiUrl()}${path[0] === "/" ? "" : "/"}${path}`;
```

Comment:
I think is not necessary to have this logic about having or not the `/` character in the path since we (developers) are controlling the paths in the requests right? So it would assume that we are starting all the paths like `/my/path` as an example

Reply:
Yeah, I will rebuild the http handler